core/src/databases/database.rs

```diff
@@ -337,6 +380,7 @@ pub struct DatabaseTable {
     table_id: String,
     name: String,
     description: String,
+    schema: Option<TableSchema>,
```

Comment:
DatabaseSchemaTable is a DatabaseTable and a schema but here DatabaseTable can have a schema.

Let's have only one of the two otherwise it's quite confusing?

Reply:
I thiiiink we should just kill DatabaseSchemaTable with fire?

Reply:
Awesome. Was able to get rid of it. 
Because of that, the `get_schema` endpoint also doesn't make any sense (it's just the same as listing tables).
So that is gone as well (which removes some `front` code too). 
Maybe some day we'll want to bring it back to return the DBML but doesn't seem needed for now